lib/copyable/copyable_extension.rb

```diff
@@ -59,6 +59,13 @@ def copyable(&block)
               # for this brand new model, visit all of the associated models,
               # making new copies according to the instructions in the copyable
               # declaration
+
+              if options[:skip_associations].present?
+                # To skip an association, change its advice to :do_not_copy
+                options[:skip_associations].each do |association|
+                  main.association_list[association] = :do_not_copy if main.association_list.keys.include?(association)
+                end
+              end
```

Comment:
That's a good question. Since it operates on `main.association_list` I'd need to pass that as well as `options[:skip_associations]` as arguments. Not impossible but possibly messy.